presto-main/src/main/java/com/facebook/presto/server/protocol/QueryResourceUtil.java

```diff
@@ -50,10 +54,13 @@
 import static com.facebook.presto.client.PrestoHeaders.PRESTO_SET_SCHEMA;
 import static com.facebook.presto.client.PrestoHeaders.PRESTO_SET_SESSION;
 import static com.facebook.presto.client.PrestoHeaders.PRESTO_STARTED_TRANSACTION_ID;
+import static com.google.common.base.Strings.isNullOrEmpty;
 import static java.lang.String.format;
 
 public final class QueryResourceUtil
 {
+    public static final String X_PRESTO_PREFIX_URI = "X-Presto-Prefix-Uri";
```

Comment:
Could we have this header defined in PrestoHeaders as the other ones used in this file ?

Reply:
Good idea. I've moved it to PrestoHeaders now.